<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAttakView</class>
 <widget class="QWidget" name="EditAttakView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="all_values_lay">
     <item>
      <layout class="QFormLayout" name="values_form_lay">
       <item row="0" column="0">
        <widget class="QLabel" name="name_label">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="name_lineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="target_label">
         <property name="text">
          <string>Cible</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="type_label">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="reach_comboBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="duration_label">
         <property name="text">
          <string>Durée</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="duration_spinBox"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="rage_aggro_label">
         <property name="text">
          <string>Rage aggro</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="rage_aggro_spinBox"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="mana_cost_label">
         <property name="text">
          <string>Coût mana</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="mana_cost_spinBox"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="photo_label">
         <property name="text">
          <string>Photo</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="photo_comboBox"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="target_comboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="EffectView" name="effect_widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="setup_lay">
     <item>
      <widget class="QPushButton" name="new_atk_button">
       <property name="text">
        <string>Nouvel Attaque</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="atk_list_view"/>
     </item>
     <item>
      <widget class="QLabel" name="img_char">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../resources/resource.qrc">:/Seb.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttons_lay">
       <item>
        <widget class="QPushButton" name="apply_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Appliquer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EffectView</class>
   <extends>QWidget</extends>
   <header location="global">effectview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
